playground/entry.js

```diff
@@ -1,14 +1,15 @@
 import { createNomen } from 'nomen-js'
 import { arrowJS } from 'nomen-js/arrow'
 import { preact } from 'nomen-js/preact'
+import { hmr } from 'nomen-js/hmr'
```

Comment:
Doesn't need to be exported as it would be one of the core modules

Reply:
Nice, I thought it'd be an external module, that's good, I'll update it.